packages/html/src/pager/templates/pager-normal.tsx

```diff
@@ -0,0 +1,7 @@
+import Pager from "../pager.spec";
+
+export const PagerNormal = (props) => (
+    <Pager
```

Comment:
This could be on a single row - https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/combobox/templates/combobox-normal.tsx#L3